tools/iree-minimal-invoke-module-main.c

```diff
@@ -0,0 +1,112 @@
+// Copyright 2022 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+// Minimal C program linking in the IREE runtime. Does not even take enough
+// command-line parameters to be able to successfully run most modules. Only
+// useful for code size/symbols monitoring purposes.
```

Comment:
I don't think `tools/` is the right place for a program we're not expecting developers/users to run. Maybe a subfolder like https://github.com/iree-org/iree/tree/main/tools/android or a new 'sample' in https://github.com/iree-org/iree/tree/main/samples (or in the other repo: https://github.com/iree-org/iree-samples/tree/main/cpp )